Compile error if target=release

**Operating system or device - Godot version:**
compile on Windows 10 with MSVC 14. Godot current head (b9e8bddf19a277b6a5302ec1e12ace780ed86b12)

**Issue description** (what happened, and what was expected):
```
core\variant_call.cpp(1196): error C2039: 'arg_names': '_VariantCall::FuncData' is not member.
core\variant_call.cpp(1209): error C2039: 'return_type': '_VariantCall::FuncData' is not member.
core\variant_call.cpp(1211): error C2039: 'return_type': '_VariantCall::FuncData' is not member.
```

**Steps to reproduce:**
scons platform=windows tools=no **target=release**
scons platform=android tools=no **target=release**

--- a/core/variant_call.cpp
+++ b/core/variant_call.cpp
@@ -54,10 +54,10 @@ struct _VariantCall {
 		int arg_count;
 		Vector<Variant> default_args;
 		Vector<Variant::Type> arg_types;
-
-#ifdef DEBUG_ENABLED
 		Vector<StringName> arg_names;
 		Variant::Type return_type;
+
+#ifdef DEBUG_ENABLED
 		bool returns;
 #endif
 		VariantFunc func;

